setup.cfg

```diff
@@ -5,7 +5,7 @@ license_files = LICENSE
 universal = 1
 
 [flake8]
-ignore = E203, E266, E501, W503, E731, B903
```

Comment:
This flake8 config is meant to match the recommended config for black: https://github.com/psf/black/blob/master/.flake8 (except with the max-line-length bumped up a bit). So `ignore` should not be edited. Can you revert this?

Reply:
I believe your link is not black's recommendation, it is simply what black project itself happens to use. The minimal black complete config can be found here https://github.com/psf/black#line-length and is simply `extend-ignore = E203`. W503 is disabled by default in flake8 so when using `extend-ignore`, only E203 needs to be ignored (plus any non-black-related ignores specific to marshmallow).

I can of course still revert if you like that better.

Reply:
Ah, you're right. Thanks for clarifying